packages/frontend/core/src/components/affine/affine-error-boundary/error-basic/info-logger.tsx

```diff
@@ -18,17 +16,17 @@ export const DumpInfo = (_props: DumpInfoProps) => {
   const currentWorkspace = useLiveData(
     useService(CurrentWorkspaceService).currentWorkspace
   );
-  const currentPageId = useAtomValue(currentPageIdAtom);
+  const currentPage = useServiceOptional(Page);
```

Comment:
we should call it `useOptionalService` or `useService(S, { optional: true})`

[![confused](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/9403235f-2f1e-48f1-a6fc-2a3a04a41ea4/confused.gif)](https://app.graphite.dev/macros/toeverything)

Reply:
`useService(S, { optional: true })` is hard in typescript...

i will use useOptionalService

Reply:
emm I feel there is no difference, may be no need to change it.